torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -778,11 +778,12 @@ class ShapePropagator {
         [this](Node* node) -> type_vec_t {
           if (auto maybe_tensor_types =
                   gatherTensorTypes<DimensionedTensorType>(node)) {
-            AT_ASSERT(maybe_tensor_types->size() == 2);
+            AT_ASSERT(maybe_tensor_types->size() >= 2);
```

Comment:
relaxing the tensor_types size contraint. This is because we have a `lerp` overload in the schema that have three tensor arguments, and shape analysis should not fail in this case. Not sure why this failure not been caught by #18811, I guess it's because test_jit_fuser.py is not triggered properly before. cc @jamesr66a